src/ci/docker/scripts/rfl-build.sh

```diff
@@ -10,14 +10,14 @@ LINUX_VERSION=v6.14-rc3
 
 # Install rustup so that we can use the built toolchain easily, and also
 # install bindgen in an easy way.
-curl --proto '=https' --tlsv1.2 -sSf -o rustup.sh https://sh.rustup.rs
-sh rustup.sh -y --default-toolchain none
+#curl --proto '=https' --tlsv1.2 -sSf -o rustup.sh https://sh.rustup.rs
+#sh rustup.sh -y --default-toolchain none
 
 source /cargo/env
 
 BUILD_DIR=$(realpath ./build)
-rustup toolchain link local "${BUILD_DIR}"/x86_64-unknown-linux-gnu/stage2
-rustup default local
+#rustup toolchain link local "${BUILD_DIR}"/x86_64-unknown-linux-gnu/stage2
+#rustup default local
```

Comment:
Wouldn't you need to do `export RUSTC="${BUILD_DIR}"/x86_64-unknown-linux-gnu/stage2/bin/rustc` as replacement?

Reply:
Probably :laughing: I'm trying to figure out how to pass the correct rustc to RfL, because I forgot it.